src_tests/unittests/osm_configurator/view/states/test_state_manager.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+import src.osm_configurator.view.states.main_window as main_window_i
+import src.osm_configurator.view.states.state_manager as state_manager_i
+import src.osm_configurator.view.states.state_name_enum as state_name_enum_i
+
+import src_tests.manualtests.osm_configurator.view.controller_stub.export_controller_stub as exportcontroller_stub_i
```

Comment:
Should we still be using the stubs? Wouldn't it be better to use the actual implemented controllers?